apps/financial-aid/web-osk/src/components/Spouse/SpouseInfo.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { ChangeEvent, useCallback, useContext, useState } from 'react'
+import { Input, Checkbox, Box, Text } from '@island.is/island-ui/core'
+
+import { UserContext } from '@island.is/financial-aid-web/osk/src/components/UserProvider/UserProvider'
+import {
+  focusOnNextInput,
+  isEmailValid,
+  sanitizeNationalId,
+} from '@island.is/financial-aid/shared/lib'
+
+interface Props {
+  hasError: boolean
+  acceptData: boolean
+  setAcceptData:
+    | ((event: React.ChangeEvent<HTMLInputElement>) => void)
+    | undefined
+  removeError: React.Dispatch<React.SetStateAction<boolean>>
+}
+
+const SpouseInfo = ({
+  hasError,
+  acceptData,
+  setAcceptData,
+  removeError,
+}: Props) => {
+  const { user, setUser } = useContext(UserContext)
+
+  return (
+    <>
+      {user && (
+        <>
+          <Box marginBottom={[2, 2, 3]} marginTop={[1, 1, 0]}>
```

Comment:
Very nitty.... But usually I like to have each element just push down or up. Here this one is creating margin top and bottom. You could add the marginTop to the fragment above and change it to a Box component, feel free to ignore if you want 🙈

Reply:
I usually do that too, however in this case above this element is radiobutton container, and margin bottom there in mobile is always 8px because error message is usually displayed right under. Now I added spouseinfo in between radiobutton container and error message, so I decided to add the extra margin because between inputs and radiobutton is 16px 😅